mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -866,19 +866,23 @@ where
 
                         let len = internal_id_hash.len();
 
-                        // TODO: Remove this at Q3 2025.
+                        // TODO: Remove this block at Q3 2025.
                         {
                             let old_internal_id_hash: [u8; 32] = internal_id_hash
                                 .get(..32)
                                 .and_then(|slice| slice.try_into().ok())
                                 .unwrap_or_default();
 
-                            let old_internal_id = old_internal_id_hash.to_vec().into();
+                            let old_internal_id: BytesJson =
+                                H256::from(old_internal_id_hash).reversed().to_vec().into();
+                            let old_internal_id_for_fees: BytesJson = H256::from(old_internal_id_hash).to_vec().into();
```

Comment:
Q: having the fee tx with reverse internal id. is that a standard thing or just a way to link the two transactions to each other?